src/main/java/uk/gov/hmcts/reform/sscs/service/HmcHearingPartiesNotifiedApi.java

```diff
@@ -17,14 +15,13 @@ public interface HmcHearingPartiesNotifiedApi {
 
     String SERVICE_AUTHORIZATION = "ServiceAuthorization";
     String PARTIES_NOTIFIED_ENDPOINT = "/partiesNotified";
-    String ID = "id";
+    String ID = "hearingID";
```

Comment:
this value is wrong, should be "id" according to the swagger spec

https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/hmc-cft-hearing-service.json#/parties-notified-controller/getPartiesNotifiedUsingGET